src/common/SourceFilter/SourceFilter.js

```diff
@@ -1,16 +1,14 @@
 import React from 'react';
 
-import OrdersSelectionFilter from '../OrdersSelectionFilter';
+import { sourceOptions } from '@folio/stripes-acq-components/util/constants';
 
-const SourceFilter = (props) => {
-  const options = [{ value: 'FOLIO', label: 'FOLIO' }];
+import OrdersSelectionFilter from '../OrdersSelectionFilter';
 
-  return (
-    <OrdersSelectionFilter
-      {...props}
-      options={options}
-    />
-  );
-};
+const SourceFilter = (props) => (
```

Comment:
basically, Source filter is also a common component, it will be used in ui-invoice